package.json

```diff
@@ -37,6 +37,7 @@
     ]
   },
   "dependencies": {
+    "chroma-js": "^1.3.7",
```

Comment:
OK, I'll move it and remove the caret.

I think once this PR is merge we could check if the chroma is really needed as I think the effect is almost not visible. It's blending the color of the gear icon.
But maybe we can use the blending elsewhere as I think this can be a nice effect if it is more visible.